pkg/apis/crd/v1alpha1/types.go

```diff
@@ -197,6 +200,10 @@ type TCPHeader struct {
 
 // Packet includes header info.
 type Packet struct {
+	SrcIP string `json:"srcIP,omitempty"`
+	DstIP string `json:"dstIP,omitempty"`
+	// Length is the IP packet length (includes the IPv4 or IPv6 header length).
+	Length uint16 `json:"length,omitempty"`
```

Comment:
What are these 3 properties used for?

Reply:
They are for reporting information of the captured packet. See: https://github.com/vmware-tanzu/antrea/pull/2029#issuecomment-813160243

But I feel later we can support them for generating the injected packets too.